chalk-integration/src/query.rs

```diff
@@ -172,20 +172,22 @@ fn environment(db: &impl LoweringDatabase) -> Result<Arc<ProgramEnvironment>, Ch
 
     let builder = &mut ClauseBuilder::new(db, &mut program_clauses);
 
```

Comment:
This function (`environment`), from what I can tell is a bit of a relic. It's only used `bench` and in the REPL (when you call "lowered"). It can probably just be removed (along with `bench` tbh). But that's a separate PR.